open_humans/templates/member/my-member-connected-data.html

```diff
@@ -4,30 +4,36 @@
 {% load private_sharing %}
 {% load utilities %}
 
-{% block head_title %}Research data{% endblock %}
+{% block head_title %}Connected data{% endblock %}
 
 {% block dashboard_main %}
+
 <div class="panel panel-default pad-all-sides">
-  <p><span class="h3">Your research data</span></p>
-  <p>Below is a summary of your data files in Open Humans.</p>
+  <h2>Connected data</h2>
+
   <p>
-    You can click on a particular data source to get options for managing your
-    incoming data and sharing it with projects.
+    You have connected to the following activities that add data to your
+    account. Click on a project to visit its activity page and learn
+    more about it, deauthorize it, or send a message to project
+    coordinators.
```

Comment:
Maybe it's already taken care of, but I couldn't figure it out by reading the code: What happens if I disconnect from a data source but the files remain in my account. Will it be listed in my data sources here? If yes, the description here is a bit misleading.

Reply:
Oh... yes, you're right, they will also be listed here. Anything that is classified as a "data source" and/or has files will show up.

Reply:
In that case it would probably be good to split the view into `active projects` on the top and `data from disconnected projects` on the bottom?

Reply:
Ah, yes, that makes sense, will do!